### PR TITLE
Update de.po

### PR DESCRIPTION
**Pull request by driet**
Merged with actual en.po with poedit, not manually. Hope, this didn't broke the file, there were some errors while importing (en.po non-standard?). Added some new strings.

### CODE REVIEW

File: src/lang/_strings/de.po [hunk 1]
```diff
@@ -56,36 +58,52 @@ msgid "Ignore terms such as 'The' and 'A' when sorting lists"
...
 
 msgctxt ""
 msgid "Enable support for reverse proxy."
 msgstr "Unterstützung für einen 'reverse proxy' einschalten."
 
 msgctxt ""
 msgid "Web settings saved."
-msgstr "Web-Einstellungen gespeichert."
+msgstr ""
+"#-#-#-#-#  0catalog.po (XBMC Addons)  #-#-#-#-#\n"
```
Comment (jez500) [inline src/lang/_strings/de.po]: This doesn't look right?

File: src/lang/_strings/de.po [hunk 2]
```diff
@@ -355,19 +384,27 @@ msgstr "Alle Serien"
 
 msgctxt ""
 msgid "Web settings"
-msgstr "Web-Einstellungen"
+msgstr ""
+"#-#-#-#-#  0catalog.po (XBMC Addons)  #-#-#-#-#\n"
```
Comment (jez500) [inline src/lang/_strings/de.po]: and here

File: src/lang/_strings/de.po [hunk 3]
```diff
@@ -355,19 +384,27 @@ msgstr "Alle Serien"
...
+"Web-Einstellungen"
 
 msgctxt ""
 msgid "Kodi settings"
-msgstr "Kodi-Einstellungen"
+msgstr ""
+"#-#-#-#-#  0catalog.po (XBMC Addons)  #-#-#-#-#\n"
```
Comment (jez500) [inline src/lang/_strings/de.po]: this one too

File: src/lang/_strings/de.po [hunk 4]
```diff
@@ -614,3 +656,39 @@ msgstr "Fortsetzen ab"
...
+msgid "Keyboard"
+msgstr "Tastatur"
+
+#~ msgctxt ""
+#~ msgid "Now playing playlists"
+#~ msgstr "Spiele Wiedergabelisten ab"
+
+#~ msgctxt ""
+#~ msgid "Vibrant Headers"
+#~ msgstr "Farbige Überschriften"
```
Comment (jez500) [inline src/lang/_strings/de.po]: these last two strings should probably be removed rather than commenting out

**Comment by jez500** [comment]
Thanks heaps @driet - looks like a couple of things seem incorrect, possibly added by poedit. 

If you could also post the errors in en,po (or patch to fix) it would be great to get that one in a clean state so there is no errors next time

cheers!

**Comment by driet** [comment]
I'll have to look into this. Working with poedit is more comfort, but seems to break some lines. Sorry for your effords.

**Pull request closed without merging**